commons/src/types/ScheduleEditor.ts

```diff
@@ -47,4 +41,20 @@ export interface Manifest extends NylasManifest {
   max_book_ahead_days: number;
   min_book_ahead_days: number;
   custom_fields: CustomField[];
+  events: any[]; // TODO
+}
+
+export interface EventDefinition {
+  event_title: string;
+  event_description: string;
+  event_location: string;
+  event_conferencing: string;
+  slot_size: 15 | 30 | 60;
+  email_ids: string[];
+  host_rules: HostRules;
+}
+
+interface HostRules {
+  method: "all" | "user_determined" | "random";
```

Comment:
What do each of these options do?

Reply:
If you have multiple hosts for a meeting definition, for a given booked meeting:
- "all" will include all hosts
- "random" will randomly assign one of them
- ""user_determined" will let the booking user choose from among them